infra/repo_manager_test.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test the functionality of the RepoManager class
+The will consist of the following functional tests
+  1. Cloning of directory in desired location
+  2. Checking out a specific commit
+  3. Can get a list of commits between two SHAs
+"""
+
+from RepoManager import RepoManager
+from RepoManager import RepoManagerError
+import os
+import unittest
+
+
+class TestRepoManager(unittest.TestCase):
```

Comment:
We should probably add some unit (ie: non-integration) tests since they will be faster and easier to debug.